railties/test/commands/secrets_test.rb

```diff
@@ -17,8 +17,21 @@ def teardown
     assert_match "No $EDITOR to open decrypted secrets in", run_edit_command(editor: "")
   end
 
+  test "edit secrets" do
+    run_setup_command
+    run_edit_command(editor: "ex -c :1d -c %s/#//g -c wq")
+    output = Dir.chdir(app_path) do
+      `bin/rails runner "puts Rails.application.secrets.external_api_key" -e production`
+    end
+    assert_equal "1466aac22e6a869134be3d09b9e89232fc2c2289…\n", output
```

Comment:
Thanks for your comment. 

I confirmed that can write to the file using `echo`(e.g. `run_edit_command(editor: "eval echo production:\\napi_key: 123 > ")`). But I agree that simulating with `cat` is enough. So I updated it to use with `cat`.